virttest/iscsi.py

```diff
@@ -210,6 +210,7 @@ def __init__(self, params, root_dir):
         self.export_flag = False
         self.luns = None
         self.iscsi_lun_attrs = params.get("iscsi_lun_attrs")
+        self.iscsi_backend = params.get("iscsi_backend", "fileio")
```

Comment:
The `class _IscsiComm` is upper logical class, The value "fileio" just is available in specifical situations such as the subclass `IscsiLIO` in other words it is an option value in the subclass `IscsiLIO`, but not in the subclass `IscsiTGT` and other iscsi utility, I mean it can not represent the common, so I think we should take a common value name to define this part then use specific option value in the command line of the underlying class depends on the upper type fo `iscsi_backend_type`.
BTW, here we could also get the type of `emulated_image` then decide how to do it next and it will not introduce a new parameter
e.g:
```
emulated_image = params.get("emulated_image")
if emulated_image:
    if emulated_image.startswith("/dev/"):
        # block backend 
    else
        # file backend
```

Reply:
Thank your suggestion, using mentioned policy if user not specify iscsi_backend in cfg file.